ci-wheel.Dockerfile

```
@@ -119,6 +119,25 @@ mv gh_*/bin/gh /usr/local/bin
 rm -rf gh_*
 EOF
 
+# Download, build, and install aws-sdk-cpp
+ARG AWS_SDK_CPP_VER=notset
+RUN <<EOF
+pushd tmp
+git clone --recurse-submodules -b ${AWS_SDK_CPP_VER} https://github.com/aws/aws-sdk-cpp.git
+cd aws-sdk-cpp
+cmake \
+  -S . \
+  -B build \
+  -DCMAKE_INSTALL_PREFIX=/usr/local \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DBUILD_ONLY=s3 \
+  -DBUILD_SHARED_LIBS=OFF \
+  -DENABLE_TESTING=OFF
```

```suggestion
  -DENABLE_TESTING=OFF \
  -DENABLE_UNITY_BUILD=ON
```

What do you think about enabling this option?

From https://github.com/aws/aws-sdk-cpp/blob/af521afd75449b7617cb27e8fff6adf67e35d613/CMakeLists.txt#L31

> _"If enabled, the SDK will be built using a single unified .cpp file for each service library.  Reduces the size of static library binaries on Windows and Linux"_

That seems useful for our purposes.

---

Good idea. I've turned on unity build - we'll see how well it works.